specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/stable/2023-01-31/digitaltwins.json

```diff
@@ -2207,8 +2207,16 @@
         },
         "recordPropertyAndItemRemovals": {
           "description": "Specifies whether or not to record twin / relationship property and item removals, including removals of indexed or keyed values (such as map entries, array elements, etc.). This feature is de-activated unless explicitly set to 'true'. Setting this property to 'true' will generate an additional column in the property events table in ADX.",
-          "type": "boolean",
-          "default": false,
+          "enum": [
+            "true",
```

Comment:
This is not a good practice to use 'true' and 'false' as enum values. Its very hard to add other meaningful enum values when you use booleans as enum values. Try to come up with another meaningful, extensible way to model what you want to configure. E.g. instead of "recordPropertyAndItemRemovals is one of (true, false)" how about "trackRemovals" is one of (none, itemsAndProperties)"